### DISCUSSION TITLE
Event Calendar

### DISCUSSION THREAD

**Discussion by pierreg-vales** [uprm-inso4101-2024-2025-s1/semester-project-campus-resource-and-event-management#11] [Ideas]
Maybe there can be a section for a calendar where organizations events appear. Maybe people can add these events to their own calendar or decide to add an event to their notifications through here.

**Comment by AngelUPR** [comment]
Do you mean kind of like the portal where you can view your classes in like a calendar view?

**Reply by pierreg-vales** [reply]
Yes very similar to that, where people can have a visual representation of events

**Comment by estefania-2817** [comment]
Will there be a filter to show which events appear on the calendar so it does not look crowded or clustered?

**Reply by Ojani** [reply]
I think that only the events you have marked to attend should be on your personal calendar. you could also have a general calendar in which you show all event, filtering by tags.

**Reply by estefania-2817** [reply]
But what if I want to discover new events related to a specific tag or topic? I believe a filter with tags could be a solution.

**Reply by Ojani** [reply]
Perhaps you didn't see the edit I made on the reply, I think the second "general calendar" solves that issue.

**Reply by estefania-2817** [reply]
You were right, I did not see the edit. I agree with your solution.
